src/core/accessibility.js

```diff
@@ -1,3 +1,66 @@
+import { isMac } from '../helpers/device';
+
 export function isKeyboardNav() {
   return document.documentElement.getAttribute('data-tabbing') === 'true';
 }
+
+export function findFocusableIndex(elements, toFind) {
```

Comment:
Kind of sad that we aren't able to use `findIndex` here :(

Reply:
It's kind of surprising that we can use `forEach` on a HTML element collection though... :0

Reply:
Yeah kinda a shame, I will implement a for loop instead of forEach though to make it slightly more efficient.